src/DefaultBackground.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { withStyles, withStylesPropTypes } from 'react-with-styles';
+import {
+  BACKGROUND_HEIGHT_UNITS,
+  HORIZONTAL,
+  VERTICAL,
+} from './constants/SliderConstants';
+import OrientationPropType from './propTypes/OrientationPropType';
+
+const propTypes = {
+  ...withStylesPropTypes,
+  // the orientation
+  orientation: OrientationPropType,
+};
+
+const defaultProps = {
+  orientation: HORIZONTAL,
+};
+
+function DefaultBackground({
+  css,
+  orientation,
+  styles,
+}) {
+  return (
+    <div
+      {...css(
+        styles.DefaultBackground_background,
+        orientation === VERTICAL ? styles.DefaulBackground_background__vertical : styles.DefaulBackground_background__horizontal
+      )}
+    />
+  );
+}
+DefaultBackground.propTypes = propTypes;
+DefaultBackground.defaultProps = defaultProps;
+
+export default withStyles(({ color, unit }) => ({
+  DefaultBackground_background: {
+    background: color.accent.lightGray,
+    border: '1px solid red',
```

Comment:
we talked about moving this variables (esp things like color) over to the theme, so just making a note of it. :)